clang/lib/DPCT/SaveNewFiles.cpp

```diff
@@ -198,6 +196,25 @@ void rewriteFileName(std::string &FileName, const std::string &FullPathName) {
 
 static std::vector<std::string> FilesNotInCompilationDB;
 
+std::map<std::string, std::string> OutFilePath2InFilePath;
+
+static bool checkOverwriteAndWarn(const std::string &OutFilePath,
+                                  const std::string &InFilePath) {
+  auto SrcFilePath = OutFilePath2InFilePath.find(OutFilePath);
+
+  bool Overwrites = false;
+  // Make sure that the output file corresponds to a single and unique input
+  // file.
+  if (SrcFilePath != OutFilePath2InFilePath.end() &&
+      SrcFilePath->second != InFilePath) {
+    llvm::errs() << "[WARNING]: Due to name collusion with another file, skip "
```

Comment:
I think we can tell the user the two input file path which cause the collusion

Reply:
Updated the message